coremltools/converters/_converters_entry.py

```diff
@@ -371,9 +371,9 @@ def skip_real_div_ops(op):
         Defaults to ``False``.
 
     compute_units: coremltools.ComputeUnit
+        The set of processing units the model can use to make predictions.
```

Comment:
--> `The set of processing units the model can use to make predictions. After conversion, the model is loaded with the provided set of compute units and returned. `